scripts/src/middleware.ts

```diff
@@ -69,6 +70,11 @@ export const notFoundHandler = function(req: Request, res: Response) {
 export function generalErrorHandler(err: any, req: Request, res: Response, next: express.NextFunction) {
 	if (err instanceof MarketplaceError) {
 		clientErrorHandler(err, req as express.Request, res);
+	} else if (err instanceof HttpError) {
```

Comment:
But what about 5xx, won't they will also be an instance of http-errors? also, what if express changes dependency? (where did you see that errors are of this type?). There must be a better way to test here. Why not for the error status being of 4xx?